source/common/unified_memory_allocation/include/uma/memory_provider_ops.h

```diff
@@ -49,6 +49,7 @@ struct uma_memory_provider_ops_t {
                                            size_t *pageSize);
     enum uma_result_t (*purge_lazy)(void *provider, void *ptr, size_t size);
     enum uma_result_t (*purge_force)(void *provider, void *ptr, size_t size);
+    void (*name)(void *provider, const char **ppName);
```

Comment:
maybe better to rename `name` -> `get_name`?

Reply:
Right, it's better for consistency with other getter functions. Done.